fea-rs/src/compile/output.rs

```diff
@@ -1,171 +1,110 @@
 //! The result of a compilation
 
-use std::collections::{BTreeMap, HashMap, HashSet};
-
 use write_fonts::{
     dump_table,
-    read::{FontRef, TableProvider, TopLevelTable},
-    tables::{
-        layout::{FeatureParams, StylisticSetParams},
-        maxp::Maxp,
-    },
-    types::Tag,
-    FontBuilder,
+    read::TopLevelTable,
+    tables::{self as wtables, maxp::Maxp},
+    validate::Validate,
+    FontBuilder, FontWrite,
 };
 
-use super::{
-    error::BinaryCompilationError,
-    features::SizeFeature,
-    lookups::{AllLookups, FeatureKey, LookupId},
-    tables::Tables,
-    tags, Opts,
-};
+use super::{error::BinaryCompilationError, Opts};
 
 use crate::{Diagnostic, GlyphMap};
 
-/// The output of a compilation operation.
+/// The tables generated by this compilation.
+///
+/// All tables are optional, and the set of tables that are present depends
+/// on the input file.
 ///
-/// This losslessly contains all of the information parsed from the FEA necessary
-/// for generating the various OpenType tables.
-//TODO: ability to generate new errors during this final compilation step
+/// Each table is a type defined in the [`write-fonts`][] crate. The caller
+/// may either interact with these directly, or else they may use the `build`
```

Comment:
I don't see a `build` method, did you mean `to_binary`?

```suggestion
/// may either interact with these directly, or else they may use the `to_binary`
```

Reply:
good catch! It also looks like I didn't have this crate set up so that broken links in documentation make us fail CI, so let me see about that...